stock_release_channel/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Stock Release Channels",
+    "summary": "Manage workload in WMS with release channels",
+    "version": "16.0.1.0.0",
+    "development_status": "Beta",
+    "license": "AGPL-3",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "maintainers": ["sebalix"],
+    "website": "https://github.com/OCA/wms",
+    "depends": [
+        "web",
+        "sale_stock",
```

Comment:
I think we don't need to depend on sales anymore since my change here:
[e334ddc0bece05fd562714205655c809e24a6caa](https://github.com/OCA/wms/commit/e334ddc0bece05fd562714205655c809e24a6caa)

Reply:
The dependency is removed and everything is working fine.